enhancements/baremetal/an-slo-for-baremetal.md

```diff
@@ -209,8 +209,9 @@ The CBO will:
 
 - Be a new `openshift/cluster-baremetal-operator` project.
 - Publish an image called `cluster-baremetal-operator`.
-- Use CVO run-level 30, so its manifests will be applied in parallel
-  with the MAO.
+- Use CVO run-level 31, so its manifests will be applied after MAO ones. This
+  is required since MAO defines the `openshift-machine-api` namespace which
+  is required also by CBO.
```

Comment:
Keeping CBo at the same run level as MAO would require CBO to maintain its own copy of the "openshift-machine-api" namespace definition and having 2 copies of it 9in MAO and CBO) adds the extra burden to keep them in sync. The two copies going out of sync can result in instabilities and lowering CBO's run level is one way to avoid that.

Do we need to add some additional text like this to explain our decision?

Reply:
Sounds ok to me, seems to explain in a clearer way the underlying motivation

Reply:
+1 for adding the extra detail